### ISSUE TITLE
conda update numpy breaks Docker build

### ISSUE THREAD

**Issue by stklik** [jupyter/docker-stacks#892]
I use `jupyter/scipy-notebook:latest`

I think this is not really necessary, but here we go:
`docker run --rm -p 8888:8888 -v $(pwd):/home/jovyan/CREST -e JUPYTER_ENABLE_LAB=yes crest-local`

1. I add a few additional things to my Docker images using a [Dockerfile](https://github.com/stklik/CREST/blob/master/Dockerfile)
2. Trigger the build `docker build -t stklik/crest .` (also tried with `--no-cache`)
3. when `RUN conda update numpy` is executed the build fails with error 

```ERROR conda.core.link:_execute(637): An error occurred while installing package 'None'.```

This has been going on for three days. 
Does somebody observe similar issues? 
I'm trying to identify if this issue has to do with conda, with numpy, with the docker-stacks or with the things that my Dockerfile does. Any ideas where I can start investigating?

EDIT: Here's a link to the output of my Travis builds (with the full error log): https://travis-ci.org/stklik/CREST/builds/551210490

PS: I also opened a thread on the Anaconda Google Group: https://groups.google.com/a/continuum.io/forum/#!topic/anaconda/qUYvwM39QBc

**Comment by parente** [comment]
Could be related to #904 as well (i.e., time to upgrade all our libraries for compatibility with the latest and greatest builds on conda-forge.)

**Comment by romainx** [comment]
Hello,

I think this problem is related to the version of `conda` see https://github.com/conda/conda/issues/8831. Some comments say that it should be fixed with the version `4.7.10`.

Latest versions of the notebooks use the conda version `4.7.12`.

I've tried on my side and it works fine.

```bash
$ conda list | grep numpy
# numpy                     1.17.5           py37h95a1406_0    conda-forge

$ conda update numpy -y
# [...]

$ conda list | grep numpy
# numpy                     1.18.1           py37h95a1406_0    conda-forge

# It can be imported as well
$ ipython
In [1]: import numpy                                                                                                                                                                                              
In [2]: numpy.__version__                                                                                                                                                                                         
# Out[2]: '1.18.1'
```

The version `1.18.1` is the latest available on conda-forge at the time of writing.

```
Package            Current     Newest
-----------------  ----------  --------
numpy              1.17.5      1.18.1
```
if everyone agrees, I think we could close this issue.

**Comment by romainx** [comment]
@stklik We have no answer from you and it seems to be OK now, so I'm closing the issue. Feel free to open a new one if you run into problem.

Best
